libvast/vast/concept/parseable/detail/as_parser.hpp

```diff
@@ -25,7 +25,7 @@ namespace detail {
 
 // -- unary -------------------------------------------------------------------
 
-inline auto as_parser(char c) {
+constexpr inline auto as_parser(char c) {
```

Comment:
Any `constexpr` function is also `inline`.

```suggestion
constexpr auto as_parser(char c) {
```